vendor/k8s.io/kubernetes/plugin/pkg/admission/defaulttolerationseconds/admission.go

```diff
@@ -75,6 +77,12 @@ func (p *plugin) Admit(attributes admission.Attributes) (err error) {
 		return errors.NewBadRequest(fmt.Sprintf("expected *api.Pod but got %T", attributes.GetObject()))
 	}
 
+	// already scheduled pods cannot be mutated
+	if attributes.GetOperation() == admission.Update {
```

Comment:
why not just remove Update from `Handler: admission.NewHandler(admission.Create, admission.Update)`?